workers/unity/Assets/Gdk/Core/GameObjectRepresentation/GameObjectManager.cs

```diff
@@ -8,21 +8,32 @@ public class GameObjectManager
     {
         private readonly Dictionary<Entity, GameObject> entityGameObjectMapping = new Dictionary<Entity, GameObject>();
 
+        public bool HasGameObjectEntity(Entity entity)
+        {
+            return entityGameObjectMapping.ContainsKey(entity);
+        }
+
         public void AddGameObjectEntity(Entity entity, GameObject gameObject)
         {
             entityGameObjectMapping[entity] = gameObject;
         }
 
-        public void TryRemoveGameObjectEntity(Entity entity)
+        public void RemoveGameObjectEntity(Entity entity)
         {
             GameObject gameObject;
             if (!entityGameObjectMapping.TryGetValue(entity, out gameObject))
             {
+                Debug.LogErrorFormat(Errors.EntityNotFound, entity.Index);
                 return;
             }
 
             entityGameObjectMapping.Remove(entity);
             Object.Destroy(gameObject);
         }
+
+        internal static class Errors
+        {
+            public const string EntityNotFound = "Entity {0} not found.";
```

Comment:
Need more context - `Entity {0} not found when trying to remove GameObject from it.`

Reply:
Obsolete now as the error was removed.